requirements-dev.txt

```diff
@@ -1,6 +1,7 @@
 # Internationalization and Localization requirements
 -e git://github.com/edx/xblock-sdk.git@v0.1.6#egg=xblock-sdk==v0.1.6
 Django==1.11.25
+django-appconf==1.0.3
```

Comment:
It might be a good idea to add a comment in the file stating that the pin is for python 2 support (saves having to check `git blame`).

Reply:
comment added